gateway/api/permissions.py

```diff
@@ -9,4 +10,6 @@ class IsOwner(permissions.BasePermission):
     """
 
     def has_object_permission(self, request, view, obj):
+        if isinstance(obj, RuntimeJob):
+            return obj.job.author == request.user
```

Comment:
Was this left over from the API part that was removed?

Reply:
Yes, I take out.  Thanks!